src/server/system_services/master_key_manager.js

```diff
@@ -43,38 +48,68 @@ class MasterKeysManager {
 
 
     get_root_key_id() {
-        return ROOT_KEY;
+        const { NOOBAA_ROOT_SECRET } = process.env;
+        return NOOBAA_ROOT_SECRET ? ROOT_KEY : undefined;
     }
 
     is_root_key(root_key_id) {
```

Comment:
I feel that keeping this old behavior is very confusing.. maybe it's better to add here the new behavior?